modules/linear_boltzmann_solvers/c_discrete_ordinates_adjoint_solver/lbs_adj_solver.h

```diff
@@ -5,23 +5,27 @@
 #include "modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/lbs_discrete_ordinates_solver.h"
 #include "framework/math/math.h"
 
-#include "modules/linear_boltzmann_solvers/c_discrete_ordinates_adjoint_solver/response_function/lbs_adj_response_function.h"
 #include "modules/linear_boltzmann_solvers/a_lbs_solver/groupset/lbs_groupset.h"
 
 namespace opensn
 {
 namespace lbs
 {
 
+/**
+ * An adjoint discrete ordinates solver.
+ *
+ * This class provides functionality to perform all necessary modifications
+ * for an adjoint discrete ordinates solve. This includes transposing the
+ * group-to-group transfers, reversing the angles, and evaluating inner
+ * products with forward material and point sources for response evaluations.
```

Comment:
forward material: I think this is for the case when the QoI is a reaction rate. But what if I just want the average flux in a volume as a QoI? Are forward material and point sources equivalent items?

Reply:
No they are not. Currently a "forward material source" is an isotropic multi-group source that gets assigned to a material. A point source is a `PointSource` object. I thought I added distributed sources to the routine, but I guess I didn't. I will do that and update the doc.